api/controllers/NetworkController.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
       .tap(network => Promise.map(req.param('communities'), communityId =>
         Membership.hasModeratorRole(req.session.userId, communityId)
```

Comment:
it's a oneline function literal, just broken into 2 lines. So it has implicit return.